cli/sawtooth_cli/config.py

```diff
@@ -317,6 +364,29 @@ def _do_config_list(args):
         raise AssertionError('Unknown format {}'.format(args.format))
 
 
+def _get_proposals(rest_client):
+    state_leaf = rest_client.get_leaf(
+        _key_to_address('sawtooth.config.vote.proposals'))
+
+    config_candidates = ConfigCandidates()
+
+    if state_leaf is not None:
+        setting_bytes = b64decode(state_leaf['data'])
```

Comment:
If that fails, somethings wrong with the rest api.  We would have failed much earlier, because we wouldn't have parsed the JSON.  We could catch it, but I'd still see it as a developer error and probably throw an AssertionError